suite-common/formatters/src/formatters/tests/prepareCryptoAmountFormatter.test.ts

```diff
@@ -37,13 +37,22 @@ describe('CryptoAmountFormatter', () => {
             ).toBe('0.000003');
         });
 
-        it('BTC balance with symbol', () => {
+        it.each([
+            ['0.3', '0.3 BTC'],
+            ['0.3000', '0.3 BTC'],
+            ['3.000', '3 BTC'],
+            ['000.3', '0.3 BTC'],
+            ['003', '3 BTC'],
```

Comment:
These two specs should be added, and they are currently failing, see fix suggestion above..
```suggestion
            ['003', '3 BTC'],
            ['0033.3', '33.3 BTC'],
            ['0033', '33 BTC'],
```

Reply:
You are right. Made one of the cases even more paranoid.